dnf-behave-tests/features/updateinfo.feature

```diff
@@ -312,3 +312,32 @@ Examples:
     # yum compatibility
     | list bugzillas      |
     | list bzs            |
+
+
+@bz1728004
+Scenario: updateinfo show <advisory> of the running kernel after a kernel update
+   When I execute dnf with args "install kernel"
+   Then the exit code is 0
+    And Transaction is following
+        | Action        | Package                                  |
+        | install       | kernel-core-0:4.18.16-300.fc29.x86_64    |
+        | install       | kernel-modules-0:4.18.16-300.fc29.x86_64 |
+        | install       | kernel-0:4.18.16-300.fc29.x86_64         |
```

Comment:
These 3 steps seem to be the test setup, so you can use `Given I successfully execute dnf ...` and then you don't need the `Then the exit code is 0`. In theory you shouldn't need the `And Transaction is following` step either, but I guess it makes the test more robust in case someone e.g. inadvertently changes the test data.